data/json/monsterdrops/zombie_default.json

```diff
@@ -76,6 +83,7 @@
                           { "item": "gown", "prob": 10, "damage": [ 1, 4 ] },
                           { "item": "dress", "prob": 70, "damage": [ 1, 4 ] },
                           { "item": "sundress", "prob": 45, "damage": [ 1, 4 ] },
+                          { "group": "christmas_dresses", "prob": 150, "damage": [ 1, 4 ], "event": "christmas" },
```

Comment:
Same issue here.   Is there a debug test to pretend it's a current event so you can test drops or do we have to test drops once a year.

Reply:
Well, I just had to test it while changing the date of my computer haha.